build_tools/benchmarks/reporting/parse_shark_benchmarks.py

```diff
@@ -0,0 +1,308 @@
+  st = st.set_caption(title)
+  st = st.applymap(html_utils.style_performance, subset=_PERF_COLUMNS)
+  st = st.set_properties(subset=[_MODEL],
+                         **{
+                             "width": "300px",
+                             "text-align": "left",
+                         })
+  st = st.set_properties(subset=[_BASELINE],
+                         **{
+                             "width": "140",
+                             "text-align": "center",
+                         })
+  st = st.set_properties(subset=[_DIALECT, _DATA_TYPE, _DEVICE],
+                         **{
+                             "width": "100",
+                             "text-align": "center",
+                         })
+  st = st.set_properties(subset=_LATENCY_COLUMNS,
+                         **{
+                             "width": "100",
+                             "text-align": "right",
+                         })
+  st = st.set_properties(subset=_PERF_COLUMNS,
+                         **{
+                             "width": "150px",
+                             "text-align": "right",
+                             "color": "#ffffff"
+                         })
+
+  return st.to_html() + "<br/>"
+
+
+def generate_table(iree_csv,
+                   baseline_csv,
+                   shark_csv=None,
+                   shape_type="static",
+                   device="cpu",
+                   title="Benchmarks"):
+  """Generates a table comparing latencies between IREE, SHARK and a baseline.
+
+  Args:
+    iree_csv: Path to the csv file containing IREE latencies.
+    baseline_csv: Path to the csv file containing baseline latencies.
+    shark_csv: Path to the csv file containing SHARK-Runtime latencies. This is optional.
+    shape_type: Currently either `static` or `dynamic`.
+    device: Device used to run the benchmarks.
+    title: The title of the generated table.
+
+  Returns:
+    A HTML file containing the summarized report.
+  """
+  shark_df = None
+  if shark_csv is not None:
+    shark_df = pd.read_csv(shark_csv)
+    shark_df = shark_df.loc[(shark_df.shape_type == shape_type) &
+                            (shark_df.device == device)]
+
+  iree_df = pd.read_csv(iree_csv)
+  iree_df = iree_df.loc[(iree_df.shape_type == shape_type) &
+                        (iree_df.device == device)]
+
+  baseline_df = pd.read_csv(baseline_csv)
+  baseline_df = baseline_df.loc[(baseline_df.shape_type == shape_type) &
+                                (baseline_df.device == device)]
+
+  return _generate_table(iree_df, shark_df, baseline_df, title)
+
+
+def main(args):
+  """Summarizes benchmark results generated by the SHARK Tank."""
+  verison_html = "<i>IREE version: " + args.iree_version + "</i><br/>"
+  verison_html = verison_html + "<i>SHARK version: " + args.shark_version + "</i><br/>"
+  verison_html = verison_html + "<i>last updated: " + datetime.today().strftime(
+      "%Y-%m-%d") + "</i><br/><br/>"
```

Comment:
```suggestion
  verison_html = (f"<i>IREE version: {args.iree_version}</i><br/>"
                  f"<i>SHARK version: {args.shark_version}</i><br/>"
                  f"<i>last updated: {date.today().isoformat()}</i><br/><br/>")
```

Did you want newlines here? Also maybe span instead of using explicit breaks?

Reply:
Done. Yeah looking to use newlines here.